nw-firestore-schema.ts

```diff
@@ -70,7 +70,7 @@ type HackerRoles = developer | designer | hardware | product | data | business |
 
 type Engagements = facebook | instagram | twitter | medium | linkedin | event
 
-type ApplicationStatus = applied | accepted | rejected
+type ApplicationStatus = applied | accepted | rejected | waitlisted
```

Comment:
Could I sneak in this small change? Otherwise this looks good!

```suggestion
type ApplicationStatus = applied | accepted | rejected | waitlisted | inProgress
```